cmd/create/cluster/cmd.go

```diff
@@ -260,7 +261,20 @@ func initFlags(cmd *cobra.Command) {
 		"cluster-name",
 		"c",
 		"",
-		"Name of the cluster. This will be used when generating a sub-domain for your cluster on openshiftapps.com.",
+		"The unique name of the cluster. The name can be used as the identifier of the cluster."+
+			" The maximum length is 54 characters."+
```

Comment:
This is what I was initially thinking, remove the validation and let the API do it for us; however this opens up for a poor UX also the toggle behind longer name is meant to be a temporary thing until QE have run their validation and eventually all validations on the backend will be performed against a 54 chars limit.

Reply:
I'm assuming this will go into `1.2.37` which is targeted for April 4th as a release date. @machi1990 is the feature due to be enabled for everyone in `prod` by that time?

Reply:
> I'm assuming this will go into 1.2.37 which is targeted for April 4th as a release date

Yes, that's our target date. 

> is the feature due to be enabled for everyone in prod by that time?

yes, the API changes are under QE testing as we speak.

Reply:
In that case, I personally don't see a problem to bump the usage message here to directly reference the 54 character length as by the time this goes out in ROSA, it will be GA to everyone cc @ciaranRoche